client/src/styles/utils/_variables.scss

```diff
@@ -12,7 +12,7 @@ $border-radius-40: 4rem;
 $box-shadow-1: 0px 0px 25px rgba(0, 0, 0, 0.05);
 $box-shadow-2: 0px 4px 25px rgba(0, 0, 0, 0.15);
 
-$transition-time-1: 0.5s;
+$transition-time-1: 0.05s;
```

Comment:
@jmikolajczyk: `ProjectTimelineWidgetItem` component uses this transition time too, but it'll be removed in https://github.com/golemfoundation/octant/pull/429, so no harm done.